andela/ah-frontend-sparta
# #164798171 Pagination support for Articles

**what does this PR do?**
- this enables users to view a specified number of articles and go to the next page for more articles

**How should this be manually tested?**

- Fetch this branch git fetch origin ft-pagination-support-articles-164798171
- Checkout to the branch git checkout ft-pagination-support-articles-164798171
- Run npm install
- Run npm start to view the landing page 
- Login to the app on /Login
- Check out the paginated articles on the dashboard, and click next to view other articles.

**What are the relevant Pivotal Tracker stories?**

[#164798171](https://www.pivotaltracker.com/n/projects/2320796/stories/164798171)

**Screenshots**

- The First Articles on the landing page

<img width="1440" alt="Screenshot 2019-05-20 at 16 43 25" src="https://user-images.githubusercontent.com/40162086/58026114-72657d80-7b1e-11e9-8045-da0ee297245b.png">

- The Next Articles when "next" is clicked.

<img width="1433" alt="Screenshot 2019-05-20 at 16 40 37" src="https://user-images.githubusercontent.com/40162086/58025951-2ca8b500-7b1e-11e9-91a5-75e4278c5d58.png">
